resolwe_bio/tools/bigwig_chroms_to_ucsc.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/python3
+"""Change chromosome names in BigWig header from ensembel or NCBI to UCSC"""
+
+import argparse
+import os
+import pyBigWig
+import sys
+
+from os import rename
+from resolwe_runtime_utils import warning
+
+
+MAPPINGS_DIR = '/opt/chrom_mappings/assets/'
+MAPPINGS_FILES = {
+    'Homo sapiens': [
+        'GRCh38.p12_ensembl2UCSC.txt',
+        'GRCh38.p12_NCBI2UCSC.txt',
+    ],
+    'Mus musculus': [
+        'GRCm38.p6_ensembl2UCSC.txt',
+        'GRCm38.p6_NCBI2UCSC.txt',
+    ],
+    'Rattus norvegicus': [
+        'rn6_ensembl2UCSC.txt',
+    ],
+}
+
+
+def parse_arguments():
+    """Parse command line arguments."""
+    parser = argparse.ArgumentParser(description="Convert the chromosome names of a bigWig file.")
+    parser.add_argument('--infile', help="Input bigWig file")
+    parser.add_argument('--outfile', help="Output bigWig file name")
+    parser.add_argument('--species', help="Species")
+    return parser.parse_args()
+
+
+def parse_mapping_file(filename):
+    """Parse mapping file."""
+    mappings = {}
+    with open(filename) as handle:
+        for line in handle:
+            line = line.strip().split()
+            if len(line) < 2 or line[1] == "":
+                # if line does not conatin names in both sources just skip it
+                continue
+            mappings[line[0]] = line[1]
+    return mappings
+
+
+def parse_mappings(species, infile, outfile):
+    """Parse file with chromosome mappings."""
+    mappings = dict()
+    # if species doesn't have prepared mapping file the script should exit with status 0 and return BigWig file
+    # with output name and warining
+    if species not in MAPPINGS_FILES:
+        msg = ('Chromosome mappings for Species "{}" are not supported.'.format(species))
+        print(warning(msg))
```

Comment:
Shouldn't we issue an error here? If we exit with `sys.exit(0)`, this won't be caught by `re-checkrc` in `bamtobigwig.sh`.

Reply:
That is the plan. The reason is to support the genomes as Dicty, which can not be mapped and showed in UCSC genome browser.